CHANGELOG.md

```diff
@@ -1,5 +1,93 @@
 [//]: # (current developments)
 
+## 4.14.0 (2022-08-02)
+
+### Enhancements
+
+* Only star activated environment in `conda info --envs`/`conda env list`. (#10764)
+* Adds new sub-command, `conda notices`, for retrieving channel notices. (#11462)
+* Notices will be intermittently shown after running, `install`, `create`, `update`, `env create` or `env update`. New notices will only be shown once. (#11462)
+* Implementation of a new `rename` subcommand. (#11496)
+* Split `SSLError` from `HTTPError` to help resolve HTTP 000 errors. (#11564)
+* Include the invalid package name in the error message. (#11601)
+* Bump requests version (`>=2.20.1`) and drop monkeypatching. (#11643)
```

Comment:
Just merged https://github.com/AnacondaRecipes/conda-feedstock/pull/7 and https://github.com/conda-forge/conda-feedstock/pull/169.